com.abagames.noiz2sa.yaml

```
@@ -0,0 +1,89 @@
+app-id: com.abagames.noiz2sa
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: run-noiz2sa.sh
+rename-desktop-file: noiz2sa.desktop
+rename-icon: noiz2sa
+rename-appdata-file: noiz2sa.appdata.xml
+finish-args:
+- "--device=dri"
+- "--share=ipc"
+- "--socket=pulseaudio"
+- "--socket=x11"
+- "--persist=.noiz2sa.prf"
+modules:
+- shared-modules/glu/glu-9.json
+- shared-modules/SDL/SDL-1.2.15.json
+- shared-modules/SDL/SDL_mixer-1.2.12.json
+- name: bulletml
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: http://http.debian.net/debian/pool/main/b/bulletml/bulletml_0.0.6.orig.tar.gz
+    sha256: 162544e708cd8f61c944f7eac2a0403d2900d29fa9b1dc45235cfccfac8ae960
+  - type: patch
+    path: patches/bulletml/fixes.patch
+  - type: patch
+    path: patches/bulletml/bulletml_d.patch
+  - type: patch
+    path: patches/bulletml/d_cpp.patch
+  - type: patch
+    path: patches/bulletml/warnings.patch
+  - type: patch
+    path: patches/bulletml/makefile.patch
+  - type: patch
+    path: patches/bulletml/includes.patch
+  - type: patch
+    path: patches/bulletml/get-rid-of-boost.patch
+  build-commands:
+  - make -C src CXXFLAGS="$(CXXFLAGS) $(CPPFLAGS) -fPIC -fpic" LDFLAGS="$(LDFLAGS)
+    -Wl,-z,defs" libbulletml.so
+  - mkdir -p /app/include/d/ /app/include/bulletml/ /app/include/bulletml/tinyxml/
+    /app/lib
+  - cp src/bulletml.d /app/include/d/
+  - cp src/*.h /app/include/bulletml/
+  - cp src/tinyxml/*.h /app/include/bulletml/tinyxml/
+  - cp src/lib*.so* /app/lib/
+- name: noiz2sa
+  no-autogen: true
+  no-make-install: true
+  subdir: src
+  make-args:
+  - "-f"
+  - makefile.lin
+  sources:
+  - type: archive
+    url: https://master.dl.sourceforge.net/project/noiz2sa/noiz2sa%20source_binary/Noiz2sa%200.51a/noiz2sa-0.51a.tar.gz
+    sha256: a7e2b689a320b49cd87a5666b38b934528c65d5a212efca81a21fdd45c9f88e6
+  - type: patch
+    path: patches/noiz2sa/0001-Use-shared-assets-location-from-a-config-header.patch
+  - type: patch
+    path: patches/noiz2sa/0002-Remove-FoeCommand-class-namespace-from-method-in-cla.patch
+  - type: patch
+    path: patches/noiz2sa/0003-Fix-makefile.patch
+  - type: file
+    path: noiz2sa.desktop
+  - type: shell
+    commands:
+    - install -Dm644 noiz2sa.desktop /app/share/applications/noiz2sa.desktop
+  - type: file
+    path: noiz2sa.png
+  - type: shell
+    commands:
+    - install -Dm644 noiz2sa.png /app/share/icons/hicolor/64x64/apps/noiz2sa.png
+  - type: file
+    path: noiz2sa.appdata.xml
+  - type: shell
+    commands:
+    - install -Dm644 noiz2sa.appdata.xml /app/share/appdata/noiz2sa.appdata.xml
+  - type: script
+    dest-filename: run-noiz2sa.sh
+    commands:
+    - !/bin/sh -e
+    - HOME=$XDG_CONFIG_HOME /app/bin/noiz2sa
+  build-commands:
+  - install ./noiz2sa /app/bin/noiz2sa
+  - install -m 555 ../run-noiz2sa.sh /app/bin/run-noiz2sa.sh
```

```suggestion
    - HOME=$XDG_CONFIG_HOME /app/bin/noiz2sa.bin
  build-commands:
  - install ./noiz2sa /app/bin/noiz2sa.bin
  - install -m 555 ../run-noiz2sa.sh /app/bin/run-noiz2sa
```

That way you don't have to change the executable in the .desktop, or the `command` in the manifest.